modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/client/ItThinClientComputeTest.java

```diff
@@ -668,6 +677,42 @@ void testBigDecimalPropagation(String number, int scale) {
         assertEquals(expected, res);
     }
 
+    @Test
+    void testExecuteMapReduce() throws Exception {
+        TaskExecution<String> execution = client().compute().submitMapReduce(List.of(), MapReduceNodeNameTask.class.getName());
+
+        assertThat(execution.resultAsync(), willBe(allOf(containsString("itcct_n_3344"), containsString("itcct_n_3345"))));
```

Comment:
`itcct_n_3344` should not be used directly. Please, get the node name from the client and then compare.